storage_controller/src/service/safekeeper_service.rs

```diff
@@ -35,7 +43,39 @@ pub struct TimelineLocateResponse {
     pub new_sk_set: Option<Vec<NodeId>>,
 }
 
+#[derive(serde::Serialize, serde::Deserialize, Clone)]
+pub struct TimelineSafekeeperMigrateRequest {
```

Comment:
this should be in the api crate, no?

Reply:
I moved it to `libs/pageserver_api/src/controller_api.rs`, but I'm not sure it's the best place:
`TimelineSafekeeperMigrateRequest` is the storcon-only request. It doesn't make any sense in `pageserver_api` crate.

With this struct it doesn't matter, but if we move `TimelineLocateResponse` to `pageserver_api` too, it will create a `pageserver_api` -> `safekeeper_api` dependency, which doesn't feel right.

WDYT? Maybe we need to introduce a `storcon_api` crate?